src/main/java/seedu/souschef/logic/commands/ListCommand.java

```diff
@@ -25,6 +33,17 @@ public ListCommand(Model model) {
     @Override
     public CommandResult execute(History history) {
         requireNonNull(model);
+        Context context = history.getContext();
+        if (context.equals(INGREDIENT)) {
+            model.sort(new IngredientDateComparator());
```

Comment:
Possible to pass in the sorted model rather than sorting it in the command?

Reply:
Sorting itself(Modifying model) should be done in command classes, not before (e.g. command parser class). Then should make separate command class like sort list command but I think this is simpler and less redundant way.